### DISCUSSION TITLE
moving originals to another disk

### DISCUSSION THREAD

**Discussion by jrossign** [RhetTbull/osxphotos#809] [General]
I am moving my whole Photos library (Ventura) to another computer and I have my originals outside of the library in an Originals folder.
Can this tool be used to update the database, so I don't have to "find original" in every album ?

Thanks

**Comment by RhetTbull** [comment]
Not yet though I am working on adding that feature. I've prototyped it (see [RepairPhotosBookmarks](https://github.com/RhetTbull/RepairPhotosBookmarks)) bit it's not mature and tested enough to recommend yet.  Unfortunately, due to the way Apple has implemented the ability to keep photos outside the library, moving them is non-trivial.  It's not just a matter of updating the path in the database (which is itself trivial) but new security-scoped bookmarks (that work with App Sandboxing) need to be created and only Photos can create these.

**Comment by jrossign** [comment]
Thanks for the info.
I tried updating the DB, but I could see it's not sufficient, even on Ventura.
Really hope Apple would make our life easier, but it seems they don't want us to maintain our own Originals folder.

Looking forward to seeing this feature in your tool.

**Comment by RhetTbull** [comment]
@jrossign I've moved this to discussions as it's not really an issue. Will update here when I get the new feature implemented for repairing bookmarks to referenced files. See also #319
